integration-test/background/atb.js

```diff
@@ -1,7 +1,7 @@
 /* global dbg:false */
 const harness = require('../helpers/harness')
 const wait = require('../helpers/wait')
-const request = require('request-promise-native')
```

Comment:
just out of curiosity, what was wrong with this library?

Reply:
It was deprecated in 2020, so I figured I would replace it with the most recommended currently (node-fetch).

Reply:
Ah, good call 👍